packages/rest/src/__tests__/acceptance/caching-interceptor/caching-interceptor.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  inject,
+  Interceptor,
+  InvocationContext,
+  Provider,
+  ValueOrPromise,
+} from '@loopback/context';
+import {Request, RestBindings} from '../../..';
+
+/**
+ * Execution status
+ */
+export const status = {
+  returnFromCache: false,
+};
+
+/**
+ * An interceptor function that caches results. It uses `invocationContext`
+ * to locate the http request
+ */
+export const cachedResults = new Map<string, unknown>();
```

Comment:
The comment says "An interceptor function" but this is a key/value map. Please fix.